api/src/main/resources/workbench.yaml

```diff
@@ -680,6 +680,23 @@ paths:
           schema:
             $ref: "#/definitions/AuditBigQueryResponse"
 
+  /v1/cron/checkClusters:
+    get:
+      security: []
+      tags:
+        - offlineCluster
+        - cron
+      description: >
+        Endpoint meant to be called offline to trigger cluster checks and
+        cleanup. Enforces upgrades for older cluster deployments. May be slow to
+        execute. Only executable via App Engine cronjob.
```

Comment:
What makes this only executable via cronjob?

Do we want the option of running this ourselves? (Not sure we need to, just checking.)

Reply:
CronInterceptor picks up the "cron" tag above. We can trigger it manually via the UI. Not sure if there's a programmatic need for this anywhere.